src/inference/src/dev/plugin.cpp

```diff
@@ -112,79 +111,31 @@ ov::Any ov::Plugin::get_property(const std::string& name, const AnyMap& argument
-                        }
-                    }
-                } catch (const ov::Exception&) {
-                } catch (const InferenceEngine::Exception&) {
-                }
-                try {
-                    auto rw_properties = m_ptr->get_property(METRIC_KEY(SUPPORTED_CONFIG_KEYS), arguments)
-                                             .as<std::vector<std::string>>();
-                    for (auto&& rw_property : rw_properties) {
-                        supported_properties.emplace_back(rw_property, PropertyMutability::RW);
-                    }
-                } catch (const ov::Exception&) {
-                } catch (const InferenceEngine::Exception&) {
-                }
-                supported_properties.emplace_back(ov::supported_properties.name(), PropertyMutability::RO);
                 return supported_properties;
             }
         }
-        // Add legacy supported properties
-        if (METRIC_KEY(SUPPORTED_METRICS) == name || METRIC_KEY(SUPPORTED_CONFIG_KEYS) == name) {
-            try {
-                return {m_ptr->get_property(name, arguments), {m_so}};
-            } catch (const ov::Exception&) {
-                auto props =
-                    m_ptr->get_property(ov::supported_properties.name(), arguments).as<std::vector<PropertyName>>();
-                std::vector<std::string> legacy_properties;
-                for (const auto& prop : props) {
-                    if ((METRIC_KEY(SUPPORTED_METRICS) == name && !prop.is_mutable()) ||
-                        (METRIC_KEY(SUPPORTED_CONFIG_KEYS) == name && prop.is_mutable()))
-                        legacy_properties.emplace_back(prop);
-                }
-                if (METRIC_KEY(SUPPORTED_METRICS) == name) {
-                    legacy_properties.emplace_back(METRIC_KEY(SUPPORTED_METRICS));
-                    legacy_properties.emplace_back(METRIC_KEY(SUPPORTED_CONFIG_KEYS));
-                }
-                if (METRIC_KEY(SUPPORTED_METRICS) == name && supports_model_caching(false))
-                    legacy_properties.emplace_back(METRIC_KEY(IMPORT_EXPORT_SUPPORT));
-
-                return legacy_properties;
-            }
-        }
-        if (METRIC_KEY(IMPORT_EXPORT_SUPPORT) == name) {
+
+        if (ov::device::capabilities.name() == name) {
             try {
                 return {m_ptr->get_property(name, arguments), {m_so}};
             } catch (const ov::Exception&) {
-                if (!supports_model_caching(false))
-                    throw;
-                // if device has ov::device::capability::EXPORT_IMPORT it means always true
-                return true;
+                std::vector<std::string> device_capabilites;
+                if (supports_model_caching())
+                    device_capabilites.emplace_back("EXPORT_IMPORT");
+                // if device has ov::device::capabilities::EXPORT_IMPORT it means always true
+                return device_capabilites;
             }
         }
```

Comment:
do we need a special handling of any properties in this method? 
I suppose we can just call `return {m_ptr->get_property(name, arguments), {m_so}};` without extra conditions